images/kcp-registrar/register.sh

```diff
@@ -25,7 +25,7 @@ usage() {
     # Parameters
     printf "The following parameters need to be passed to the script:\n"
     printf "KCP_ORG: the organization for which the workload clusters need to be registered, i.e.: root:pipelines-service\n"
-    printf "KCP_WORKSPACE: the name of the workspace where the workload clusters get registered (created if it does not exist), i.e: compute\n"
+    printf "KCP_WORKSPACE: the name of the existing workspace where the workload clusters get registered, i.e: compute\n"
```

Comment:
Why is register.sh modified as part of this PR?
This is totally out of scope.
You are bending things to fit unspoken assumptions  that you have made when writing the scripts. This is not the proper way to proceed.
If you think that the assumptions made in register.sh are not correct let discuss them, let discuss, think through and document the operational model. When that's done we can make an informed decision on what makes more sense